docs/use/setup-wallet/index.md

```diff
@@ -97,38 +99,17 @@ Finally, to see an overview of your CFG accounts, click on "Receive" under your
 
 ![](./images/talisman-wallet.png#width=400)
```

Comment:
Line 74:
- Download the extension [here](https://chrome.google.com/webstore/detail/talisman-polkadot-wallet/fijngjgcjhjmmpcmkeiomlglpeiijkld/related

swap link in the post with this link https://www.talisman.xyz/wallet